### PR TITLE
Showing a message when adding to a null System Group

### PR DESCRIPTION
**Pull request by un-tone**
For https://github.com/Starcounter/CompanyTrack/issues/244

Implemented showing a warning message when adding a user to a null System Group.
I assume that  more correct  approach is to use sgb-property-metadata
component instead of simple block, but I can't find any well-working
example.

### CODE REVIEW

File: src/UserAdmin/wwwroot/UserAdmin/viewmodels/partials/administrator/EditPersonPage.html [hunk 1]
```diff
@@ -6,6 +6,11 @@
     <template is="dom-bind">
         <sgb-property-metadata items="{{model.PropertyMetadataItems}}"></sgb-property-metadata>
         <h2>{{model.FullName}}</h2>
+
+        <template is="dom-if" if="{{model.Message}}">
+            <p class$="{{model.MessageCss}}">{{model.Message}}</p>
```
Comment (Joltrix) [inline src/UserAdmin/wwwroot/UserAdmin/viewmodels/partials/administrator/EditPersonPage.html]: I may be missing something, but will `model.MessageCss` ever change from "alert alert-danger"? 

Cause if not, can't you just have it `class="alert alert-danger"` ?

Comment (un-tone) [inline src/UserAdmin/wwwroot/UserAdmin/viewmodels/partials/administrator/EditPersonPage.html]: Removed that field and binding.

**Comment by warpech** [comment]
Great solution! Please merge.

**Comment by warpech** [comment]
Great solution! Please merge.

Thanks @Joltrix for your review, good point

**Pull request merged**